components/drops/TimeTag.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div
+    class="tag-container is-flex border py-1 pl-2 pr-3 is-justify-content-space-between is-align-items-center">
```

Comment:
`py-1 pl-2 pr-3` why padding left and right are not the same?

Reply:
Don't we have a btn component that look like that?

Reply:
>py-1 pl-2 pr-3 why padding left and right are not the same?

i tried to improve the padding

they are still not equal because of the img contributing to the feeling of padding
![image](https://github.com/kodadot/nft-gallery/assets/22791238/fa19b200-d323-4054-ac97-78164c683bdd)